dev-libs/libsass/libsass-3.6.3.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools multilib-minimal
+
+if [[ ${PV} = *9999 ]]; then
+	EGIT_REPO_URI="https://github.com/sass/libsass.git"
+	inherit git-r3
+	KEYWORDS=
+else
+	SRC_URI="https://github.com/sass/libsass/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86 ~amd64-linux"
+fi
+
+DESCRIPTION="A C/C++ implementation of a Sass CSS compiler"
+HOMEPAGE="https://github.com/sass/libsass"
+LICENSE="MIT"
+SLOT="0/1" # libsass soname
+IUSE="static-libs"
+
+RDEPEND=""
+DEPEND="${RDEPEND}"
+
+DOCS=( Readme.md SECURITY.md )
+
+src_prepare() {
+	default
+
+	if [[ ${PV} != *9999 ]]; then
+		[[ -f VERSION ]] || echo "${PV}" > VERSION
+	fi
+	eautoreconf
+
+	# only sane way to deal with various version-related scripts, env variables etc.
+	multilib_copy_sources
+}
+
+multilib_src_configure() {
+	econf \
+		$(use_enable static-libs static) \
+		--enable-shared
```

Comment:
Up to you, but we'd prefer an array form here. But it's not required.

Reply:
What do you mean with 'array form'? I found this [wiki page](https://devmanual.gentoo.org/ebuild-writing/functions/src_configure/configuring/index.html) but it doesn't say anything about an alternate form.

Reply:
```
local myeconfargs=(
	$(use_enable static-libs static)
	--enable-shared
)

econf "${myeconfargs[@]}"
```
We use this form more nowadays because there have been some errors with backlashes in the past. But it's not a strict requirement and up for the maintainer until that happens.

Reply:
Aah I see, this does look a lot nicer in my opinion. I've never been very fond of escaping the line break, I find that it makes code less readable. I've changed the `src_configure` to your version.